frontend/views/reports.js

```diff
@@ -1,6 +1,8 @@
 var Backbone = require('backbone'),
     _ = require('underscore'),
     flot = require('Flot'),
+    flotcategories = require('Flot/jquery.flot.categories'),
+    flotstack = require('Flot/jquery.flot.stack'),
```

Comment:
None of these `flot` variables are actually used in this file.  From a quick look at the Flot docs, I think we just load `flot` in order to add the `plot` method to jQuery. I'm going to suggest we do this in `index.js` rather than here, maybe with a comment that loading it extends jQuery in case people wonder why it's being loaded there either.

While we're at it... we are using `$` in this file without requiring it here.